modules/uhttp/filters_test.go

```diff
@@ -42,14 +42,67 @@ import (
 	"github.com/uber/jaeger-client-go/config"
 )
 
-func TestFilterChain(t *testing.T) {
+func TestDefaultFiltersWithNopHost(t *testing.T) {
+	// setup
 	host := service.NopHost()
+	stats.SetupHTTPMetrics(host.Metrics())
+
+	t.Run("run parallel", func(t *testing.T) {
+		t.Run("testFilterChain", func(t *testing.T) {
+			t.Parallel()
+			testFilterChain(t, host)
+		})
+		t.Run("testFilterChainFilters", func(t *testing.T) {
+			t.Parallel()
+			testFilterChainFilters(t, host)
+		})
+		t.Run("testPanicFilter", func(t *testing.T) {
+			t.Parallel()
+			testPanicFilter(t, host)
+		})
+		t.Run("testMetricsFilter", func(t *testing.T) {
+			t.Parallel()
+			testMetricsFilter(t, host)
+		})
+	})
+
+	// teardown
+	httpMetricsTeardown()
+}
+
+func TestDefaultFiltersWithNopHostAuthFailure(t *testing.T) {
+	// setup
+	host := service.NopHostAuthFailure()
+	stats.SetupHTTPMetrics(host.Metrics())
+
+	t.Run("run parallel", func(t *testing.T) {
+		t.Run("testFilterChainFiltersAuthFailure", func(t *testing.T) {
+			t.Parallel()
+			testFilterChainFiltersAuthFailure(t, host)
+		})
+	})
+
+	// teardown
+	httpMetricsTeardown()
+}
+
+func TestDefaultFiltersWithNopHostConfigured(t *testing.T) {
+	// this test's sub tests cannot run parallel
+	// and they need to build host by theirselves
+	t.Run("testTracingFilterWithLogs", func(t *testing.T) {
+		t.Parallel()
```

Comment:
I put it here so when follow devs add more tests, they will follow the style and add t.Parallel in their sub tests